acceptance-tests/pom.xml

```diff
@@ -142,5 +176,106 @@
                 <maven.test.skip>true</maven.test.skip>
             </properties>
         </profile>
+        <profile>
+            <id>acceptance-tests</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <includes>
+                                <include>it/**/*.java, !*SmokeTest*.java</include>
+                            </includes>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>smoke-tests</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <configuration>
+                            <includes>
+                                <include>it/**/*SmokeTest.java</include>
+                            </includes>
+                        </configuration>
+                        <executions>
```

Comment:
Ah, no reason (anymore). When I was struggling with the combination of `bitbucket-maven-plugin` and failsafe, I thought maybe this will work better (turns out it had no effect in this case). From [the failsafe plugin docs](https://maven.apache.org/surefire/maven-failsafe-plugin/usage.html):

> For very complex builds, it may be better to separate the executions for the integration-test and verify goals.

Reply:
- [ ] combine executions